cloudify_marketplace/blueprint/configure_aws.yaml

```diff
@@ -58,6 +96,12 @@ node_templates:
           executor: central_deployment_agent
           inputs:
             user_ssh_key: { get_input: user_ssh_key }
+            manager_names_and_ips: { get_input: manager_names_and_ips }
+            broker_names_and_ips: { get_input: broker_names_and_ips }
+            new_manager_username: { get_input: new_manager_username }
+            new_manager_password: { get_input: new_manager_password }
+            new_broker_username: { get_input: new_broker_username }
+            new_broker_password: { get_input: new_broker_password }
 
   # TODO: Split this out and composite the general config part
```

Comment:
I realise they're not yours, but could you tear out the # TODO lines in the blueprints?